components/dialog/ConfirmationDialogContent.tsx

```diff
@@ -1,18 +1,19 @@
 import { DialogClose, DialogContent } from "@components/Dialog";
 import * as DialogPrimitive from "@radix-ui/react-dialog";
 import { ExclamationIcon } from "@heroicons/react/outline";
-import React from "react";
+import React, { PropsWithChildren } from "react";
+import { Button } from "@components/ui/Button";
 
-export default function ConfirmationDialogContent({
-  title,
-  alert,
-  confirmBtnText,
-  cancelBtnText,
-  onConfirm,
-  children,
-}) {
-  confirmBtnText = confirmBtnText || "Confirm";
-  cancelBtnText = cancelBtnText || "Cancel";
+export type ConfirmationDialogContentProps = {
+  confirmBtnText?: string;
+  cancelBtnText?: string;
```

Comment:
I can't see that you have any defaults - either add fallbacks or make them mandatory


```suggestion
  confirmBtnText: string;
  cancelBtnText: string;
```

Reply:
![image](https://user-images.githubusercontent.com/1046695/130830692-e57c153b-bc57-4f7e-b75d-13ac48b20a64.png)

Lint removed my newlines so it became less clear :(

Reply:
Aaahhh, ignore then!